### PR TITLE
Add Plugin Preview to GlotPress

### PR DESCRIPTION
**Pull request by trymebytes**
## Problem

There's no way to live preview the GlotPress plugin.

## Solution

This adds a "Test Preview" button to the GlotPress page in the plugin directory. When the `Test Preview` is clicked, GlotPress is loaded in a playground allowing it to be previewed right there in the browser.

The blueprint in this PR can be tested with this link: https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/GlotPress/GlotPress/add-playground-preview/.wordpress-org/blueprints/blueprint.json

### CODE REVIEW

File: assets/blueprints/blueprint.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
...route = new GP_Route_Glossary_Entry();if ( file_exists( '/wordpress/wp-content/languages/glossary-de_DE.csv' ) ) {$locale_glossary_translation_set = GP::$translation_set->by_project_id_slug_and_locale( 0, 'default', 'de' );$locale_glossary = new GP_Glossary();$locale_glossary->translation_set_id = $locale_glossary_translation_set->id;$locale_glossary = GP::$glossary->create_and_select( $locale_glossary );echo $glossary_entry_route->read_glossary_entries_from_file( '/wordpress/wp-content/languages/glossary-de_DE.csv', $locale_glossary->id, 'de' ), ' locale glossary entries', PHP_EOL;}if ( file_exists( '/wordpress/wp-content/languages/plugins/friends-glossary-de_DE.csv' ) ) {$project = GP::$project->by_path( 'local-plugins/friends' );$glossary_translation_set = GP::$translation_set->by_project_id_slug_and_locale( $project->id, 'default', 'de' );$glossary = new GP_Glossary();$glossary->translation_set_id = $glossary_translation_set->id;$glossary = GP::$glossary->create_and_select( $glossary );echo $glossary_entry_route->read_glossary_entries_from_file( '/wordpress/wp-content/languages/plugins/friends-glossary-de_DE.csv', $glossary->id, 'de' ), ' project glossary entries', PHP_EOL;;}",
```
Comment (akirk) [inline assets/blueprints/blueprint.json]: I don't think we need the glossaries for this live preview, they are just necessary for translate live.

File: assets/blueprints/blueprint.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          "permalink_structure": "/%year%/%monthnum%/%day%/%postname%/"
+        }
+      },
+      {
+        "step": "writeFile",
+        "path": "/wordpress/wp-content/mu-plugins/gp-sqlite.php",
+        "data": "<?php add_action( 'admin_footer', function() {echo '<input type=\\\"checkbox\\\" name=\\\"gp_enable_inline_translation\\\" checked=\\\"checked\\\" style=\\\"display: none\\\" />';} );add_filter( 'query', function( $query ) {return str_replace( ' BINARY ', ' ', $query );} );add_filter( 'gp_inline_translation_frontend_url', function( $url ) {return 'https://translate.wordpress.org/';} );if ( ! function_exists( 'libxml_clear_errors' ) ) {function libxml_clear_errors() {}}if ( ! function_exists( 'libxml_use_internal_errors' ) ) {function libxml_use_internal_errors() {}}if ( ! function_exists( 'libxml_get_errors' ) ) {function libxml_get_errors() {}}if ( ! function_exists( 'mysql_get_client_info' ) ) {function mysql_get_client_info() {}}if ( ! function_exists( 'mb_str_split' ) ) {function mb_str_split(  $string = '', $length = 1 , $encoding = null ){return str_split( $string, $length );}}if ( ! class_exists( 'DOMDocument' ) ) {class DOMDocument {function loadHTML() {}}}"
```
Comment (akirk) [inline assets/blueprints/blueprint.json]: Here also the filter adding the `gp_enable_inline_translation` checkbox is not necessary.

File: assets/blueprints/blueprint.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      },
+      {
+        "step": "mkdir",
+        "path": "/wordpress/wp-content/languages/plugins"
+      },
+      {
+        "step": "mkdir",
+        "path": "/wordpress/wp-content/languages/themes"
+      },
+      {
```
Comment (akirk) [inline assets/blueprints/blueprint.json]: Can be removed
```suggestion
```

File: assets/blueprints/blueprint.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        "step": "setSiteOptions",
+        "options": {
+          "WPLANG": "de_DE",
```
Comment (akirk) [inline assets/blueprints/blueprint.json]: We don't want to set the language.
```suggestion
```

**Comment by akirk** [comment]
@trymebytes Could you please test this newest version? Thanks!

**Comment by trymebytes** [comment]
Tested and works as expected, thank you!

**Comment by akirk** [comment]
I've moved the directory to `.wordpress-org` so that the https://github.com/10up/action-wordpress-plugin-asset-update action will deploy it.

**Review by akirk — APPROVED** [review]

**Pull request merged**